documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -231,9 +243,21 @@ Currently {prodname} supports the `incremental` and `blocking` types.
 | An array of comma-separated regular expressions that match the fully-qualified names of the tables to include in the snapshot. +
 Specify the names by using the same format as is required for the xref:{context}-property-signal-data-collection[signal.data.collection] configuration option.
 
-|`additional-condition`
+|`[.line-through]#additional-condition#`
+|_N/A_
+| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot. +
+ +
+  [NOTE]
```

Comment:
@roldanbob can you help me identify why this is not rendered as a `note` box? I saw it is used also in other pages.

Reply:
Got it! Many thanks!